.github/workflows/mas_check.yml

```diff
@@ -22,47 +22,55 @@ jobs:
     runs-on: ubuntu-20.04
 
     env:
-      SDL_AUDIODRIVER: dummy # handles ALSA issues
+      MAS_RENPY_VER: 8.1.1
+      MAS_DIR: mas
 
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
 
       - name: Setup Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v5
         with:
           # Version range or exact version of a Python version to use, using SemVer's version range syntax.
           python-version: 3.10.4 # optional, default is 3.x
 
-      # dl renpy src
+      # get/dl renpy src
+      - name: cache rpy source
+        id: cache-rpy
+        uses: actions/cache@v4
+        with:
+          path: renpy/
+          key: ${{ runner.os }}-rpy
+
       - name: Download rpy source
+        if: steps.cache-rpy.outputs.cache-hit != 'true'
         run: |
-          renpysdk=$(wget -qO- https://nightly.renpy.org/current-8/index.html | grep -P -m 1 -o '(?<=href=").*\.tar\.bz2(?=".*)')
-          wget https://nightly.renpy.org/current-8/$renpysdk
+          renpysdk=renpy-$MAS_RENPY_VER-sdk.tar.bz2
+          wget https://www.renpy.org/dl/8.1.1/$renpysdk
           tar xf $renpysdk
           rm $renpysdk
           mv ${renpysdk/.tar.bz2/} renpy
 
       # get/download base mas
-      # - name: cache base MAS
-      #   id: cache-mas
-      #   uses: actions/cache@v2
-      #   with:
-      #     path: mas0105/
-      #     key: ${{ runner.os }}-mas
-
-      - name: Download base MAS
-      #  if: steps.cache-mas.outputs.cache-hit != 'true'
+      - name: cache base MAS
+        id: cache-mas
+        uses: actions/cache@v4
+        with:
+          path: mas/
+          key: ${{ runner.os }}-mas
```

Comment:
Is there a reason to cache MAS per-platform? We always download the same archive. It looks like a dummy cache key that always hits after the first time, maybe we should just use a static string to indicate that? Same for renpy cache.

Reply:
its there so we we could do building on other platforms, but not necessary now so changed to simple key